### PR TITLE
ci(lint): exit with error when issues are found

### PR DESCRIPTION
**Pull request by uncenter**
@nekowinston

### CODE REVIEW

**Comment by uncenter** [comment]
The CI failing means it is working :)

**Comment by nekowinston** [comment]
Thanks, just changed some of my old bad code there :)

**Review by nekowinston — APPROVED** [review]

File: scripts/lint/stylelint.ts [hunk 1]
```diff
@@ -24,14 +22,14 @@ export const lint = async (entry: WalkEntry, content: string, fix: boolean) => {
...
-            file,
+            file: relative(REPO_ROOT, entry.path),
             startLine: warning.line,
             endLine: warning.endLine,
             startColumn: warning.column,
             endColumn: warning.endColumn,
             content,
           }, warning.severity);
         });
+        if (result.warnings.length > 0) throw new Error("stylelint error");
```
Comment (uncenter) [inline scripts/lint/stylelint.ts:33]: Why would we through here? Wouldn't it not get through all of the files?

Comment (uncenter) [reply scripts/lint/stylelint.ts:33]: @nekowinston 👀

Comment (nekowinston) [reply scripts/lint/stylelint.ts:33]: This is invoked over in
```ts
for await (const entry of stylesheets) {
  lint()
```
and we only throw after all the warnings have been printed.

**Pull request merged**